test/unittests/execute_numeric_test.cpp

```diff
@@ -163,76 +165,76 @@ TEST(execute_numeric, i64_eqz)
 
 TEST(execute_numeric, i64_eq)
 {
-    EXPECT_THAT(execute_binary_operation(Instr::i64_eq, 22, 20), Result(0));
-    EXPECT_THAT(execute_binary_operation(Instr::i64_eq, 22, 22), Result(1));
+    EXPECT_THAT(execute_binary_operation(Instr::i64_eq, 22_u64, 20_u64), Result(0));
+    EXPECT_THAT(execute_binary_operation(Instr::i64_eq, 22_u64, 22_u64), Result(1));
 }
 
 TEST(execute_numeric, i64_ne)
 {
-    EXPECT_THAT(execute_binary_operation(Instr::i64_ne, 22, 20), Result(1));
-    EXPECT_THAT(execute_binary_operation(Instr::i64_ne, 22, 22), Result(0));
+    EXPECT_THAT(execute_binary_operation(Instr::i64_ne, 22_u64, 20_u64), Result(1));
+    EXPECT_THAT(execute_binary_operation(Instr::i64_ne, 22_u64, 22_u64), Result(0));
 }
 
 TEST(execute_numeric, i64_lt_s)
 {
-    EXPECT_THAT(execute_binary_operation(Instr::i64_lt_s, 22, 20), Result(0));
-    EXPECT_THAT(execute_binary_operation(Instr::i64_lt_s, 20, 22), Result(1));
+    EXPECT_THAT(execute_binary_operation(Instr::i64_lt_s, 22_u64, 20_u64), Result(0));
+    EXPECT_THAT(execute_binary_operation(Instr::i64_lt_s, 20_u64, 22_u64), Result(1));
     EXPECT_THAT(execute_binary_operation(Instr::i64_lt_s, uint64_t(-41), uint64_t(-42)), Result(0));
```

Comment:
If negative literals are supported, these should be something like `-41_i64`? I agree keeping the explicit `unit64_t()` may be nicer, however confusing it looks to have both ways.

Reply:
The `-41_u64` is correct and can be used already. This is `-uint64_t{41}`. Although this usually case MSVC complain about applying `-` operator to an unsigned type.

Using signed types `int32_t` and `int64_t` is also an option, but I don't recall that is the current status of this support. Definitely does not look like signed types were supported when these tests were created.

There is no change here because the provided values have the correct types. And making tests look beautiful and consistent is not the goal of this PR.

Reply:
It is not about beauty, just removing confusion.